Aegis class long term care vessel(But again since VSCODE wanted to break.)

Draggeru opened the pull request:
This is literally just #1313. But because VSCODE broke in a way my noncoder brain could fix I'm re pring this on a fresh branch. For simplicity here is everything ripped from the last PR:

Adds the SSV Aegis, a roleplay-focused medical and service specialized ship to aid or take over rescue class ship cases that are above the critical care level that the rescue class level can handle, such as psychological treatment or long-term patient boarding.
## About The Pull Request

Adds another syndicate-aligned faction medical/service ship(mostly roleplay oriented), this ship was designed with not only a mid/high crew in mind but with the added potential to accept and host other non-crew members for a length of time for whatever roleplay flavor. I really need to stress this ship was designed with a primary roleplay focus, it was designed in a way where people could spend weeks and even months on this one ship alone. This is what sets the Aegis apart from the Geneva, the Geneva is more gameplay oriented in comparison. The Aegis also features a joinable psychologist role and a joinable long-term patient role. Both of these roles are roleplay oriented.

![aegisaegisaegis](https://user-images.githubusercontent.com/81540056/181124507-7acfdef8-3795-472c-9d14-6e68d28d1b78.png)

For lore: Your outstanding orders from the NSV are to respond to any major critical care situations where the patient may potentially need to be onboarded for any length of time, or in cases where first response ships have a major patient overflow. While you are not necessarily a first response vessel you are able to uphold such needs in the case of an emergency where rescue class vessels cannot respond.

The Aegis does NOT have any form of research potential, only housing a singular medical tech fab. If you somehow end up with research potential you should not spec into cargo/engineering/security. Your primary focus is to uphold NSV's orders and provide the care you specialize in.

This PR adds requested changes from maptainers after its first test merge.

## Why It's Good For The Game

More faction ship= good!

## Changelog

:cl:
add: Adds the Aegis, a ship designed primarily for medical and service(and roleplay)
:cl:

Draggeru committed:
Ah shit, here we go again

Draggeru committed:
I hate vscode so fucking much

HERE IS THE MAP FOR THE MF MAP PR.

Draggeru committed:
I hate coding I hate coding.

VSCODE YOU BITCH.

Draggeru committed:
Merge branch 'shiptest-ss13:master' into Aegis-but-again

Draggeru committed:
Delete syndicate_aegis.dmm

Wrong fucking file

Signed-off-by: Thedragmeme <81540056+Draggeru@users.noreply.github.com>

Draggeru committed:
AAAAAAAAAAAAAAAAAAAAAAAAAA

I am a shattered human being.

Draggeru committed:
Delete syndicate_aegis.dmm

Lets try this again!

Signed-off-by: Thedragmeme <81540056+Draggeru@users.noreply.github.com>

Draggeru committed:
This is vscodes fault

Draggeru committed:
Changes headsets via maptainer request

Draggeru committed:
Merge branch 'Aegis-but-again' of https://github.com/Draggeru/ShiptestDragFork into Aegis-but-again

Apogee-dev commented:
this ship has been testmerged for quite some time now and it's received a fair amount of play in that time. I don't think it has any outstanding issues remaining and it seems to have settled into a good niche as an RP-focused syndicate ship. I have no issues rubber stamping it, it just needs approval from prefab or another mapmaint and you should be good to go.

PrefabQuasar reviewed (dismissed):
this ship is great looking and feeling to play in, and does seem to direct players towards RP in a good way, but some of the equipment here is just way too much for a non-combat long term care ship
![image](https://user-images.githubusercontent.com/78622038/186526684-11bd184a-f4b4-4f09-adf1-0cb9b489b6ab.png)
this image contains the notable offenders, the stetchkin is fine as it seems to be the only gun aboard, but the captains closet could stand to have the whetstone and combat gloves plus removed, theres a LOT of nvg goggles on this ship, which doesnt seem to make too much sense for long-term care, and theres also a set of advanved med tools, which i'm iffy on, i'd suggest taking a general balance pass on the ship equipment as a whole, but i'd be satisfied with just a removal of the things mentioned above

Draggeru committed:
Fuckin vscode bullshit

Signed-off-by: Thedragmeme <81540056+Draggeru@users.noreply.github.com>

Draggeru committed:
More vscode bullshit

AAAAAAAAAAAAA.

Draggeru commented:
I went through and removed the above-mentioned major points, Ive also removed things that had duplicates such as the hyposprays. I've removed both inducers. Im hesitant to remove the advanced tools as they are the only existing set locked behind a door only the captain/lead doctor can access and the ship has a medical focus.

Draggeru commented:
Oh no there are errors now- Fuck.

Draggeru committed:
Merge branch 'master' into Aegis-but-again

Latentish commented:
Please update your floor decals and I'll give you a review.

Draggeru committed:
HHhhhh

Signed-off-by: Thedragmeme <81540056+Draggeru@users.noreply.github.com>

Draggeru committed:
Please dont break

Draggeru committed:
Vscode broke again

Signed-off-by: Thedragmeme <81540056+Draggeru@users.noreply.github.com>

Draggeru committed:
Dont break!!

Draggeru committed:
Merge branch 'shiptest-ss13:master' into Aegis-but-again

Latentish commented on _maps/configs/aegis.json:
```diff
@@ -0,0 +1,39 @@
+	},
+	"Long Term Patient": {
+		"outfit": "/datum/outfit/job/prisoner/longtermpatient",
+		"slots": 1
```
Suggestion: Raise this to 2, since the ship exists for patients.

Latentish commented on _maps/configs/aegis.json:
```diff
@@ -0,0 +1,39 @@
+		"slots": 1
+	}
+},
+"cost": 750
```
Change this to enabled = true like the other ships.

Latentish commented on _maps/configs/aegis.json:
```diff
@@ -0,0 +1,39 @@
+	},
+	"Deck Service Assistant": {
+		"outfit": "/datum/outfit/job/botanist/syndicate/nsv",
+		"slots": 3
```
Suggestion: Reduce this to 2.
The kitchen/botany on this ship could be run by 2 people easily. And with cooking appliances only being circuits, it's probably better to increase to 3 during the game if needed, rather than drawing from other ship positions.

Latentish reviewed (changes requested):
![image](https://user-images.githubusercontent.com/55075747/188796728-152106cc-e355-4d2f-8c96-7f2e420fe016.png)
This is scrungled. Please don't have anything nailed down on top of an apc terminal. Also please give the cryo pod a bit more circumstance. A good idea is to pair it with a table with basic supplies like radio/gps/flashlight. You could put it where your table in the corner of the same room is.

![image](https://user-images.githubusercontent.com/55075747/188797679-d9d0c59e-d2b2-42fc-b8ff-90f32ed84585.png)
Please explain this one to me chief. Why is the  syndicate gun and equipment bay right across from the bar? Moreover, why does an NSV inpatient ship have a weapon and equipment bay? It just feels out of place. I'd recommend putting the basic stuff, a knife and gun in the locker of the captain. The rest could be replaced with more thematically appropriate stuff like clothing vendors and other amenities for the crew.

![image](https://user-images.githubusercontent.com/55075747/188797642-de6fed97-4343-4a64-836e-2b39109e2e5a.png)
If this is a doctoring ship for doctoring people who need doctors, it'd probably be pretty neat. Even still, this isn't a very pretty way of applying dust and grime.  I'd really like to see this place a bit cleaner maybe. 

![image](https://user-images.githubusercontent.com/55075747/188797946-ea3353ea-8353-4bdf-9c6b-1eefb00abbbf.png)
Make sure to not use any tinyfans, and putting up atmos holofield projectors.

![image](https://user-images.githubusercontent.com/55075747/188798050-345ca28d-dfc1-45c7-9f02-244cdda2f3c6.png)
Luxemburg is the only exception to non-NT ships getting the captain's sabre please remove it.

![image](https://user-images.githubusercontent.com/55075747/188798405-9850d361-8b75-4ef5-beb5-1d1bd707bfc6.png)
You should instead make a new subtype of the syndicate surgical duffelbag that has normal surgery gear in it (syndicate duffelbag doesn't cause slowdown).. Otherwise a normal surgery duffelbag is fine. If they want advanced tools they can trade.

Draggeru committed:
json changes

Draggeru committed:
json more like hson lmao

Draggeru committed:
Merge branch 'Aegis-but-again' of https://github.com/Draggeru/ShiptestDragFork into Aegis-but-again

Latentish commented:
This is good. Sorry for how long it took. MAAARRRGE.
